auth/handlers/oauth2.go

```diff
@@ -186,3 +192,46 @@ func getToken(res *http.Response) (GitHubAccessToken, error) {
 
 	return token, fmt.Errorf("no body received from server")
 }
+
+func getUserOrganizations(name string) (string, error) {
+
+	organizations := []Organization{}
+	apiURL := fmt.Sprintf("https://api.github.com/users/%s/orgs", name)
+
+	req, reqErr := http.NewRequest(http.MethodGet, apiURL, nil)
+	if reqErr != nil {
+		return "", reqErr
+	}
+
+	client := http.Client{}
```

Comment:
Please use `http.DefaultClient` instead of creating a new client in this scenario.